lwtime/nattime.go

```diff
@@ -188,7 +188,7 @@ func ParseNatural(n string) (time.Time, time.Time, error) {
 	// time.Now() is intentional here such that snaps work properly
 	// For instance snapping to @d should snap to the start of the local day
 	startLocal, endLocal, err := parseNaturalFromTime(n, time.Now())
-	return startLocal.UTC(), endLocal.UTC(), err
+	return startLocal.UTC(), endLocal.UTC().Add(-time.Minute), err
```

Comment:
I think that this will remove one minute to all Natural Times, what we need is actually
modifying only the snap `now` to maybe remove 30 seconds.

@hazedav what do you think?

Reply:
From the ticket 
``` 
Command used:
lacework alert ls --range "last 3 days" --severity info --status Open --json | ConvertFrom-Json
```